.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,16 @@
+---
+name: User Story
+about: Tell us about how you are using KUDO, and get your logo on our website and
+  we'll send you some awesome custom swag !
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Name of your company
+
+## Tell us about how you are using KUDO in a couple of sentences
+
+
+Add your logo to the issue as an attachment - JPEG or PNG, and less than 10MB please
```

Comment:
```suggestion
Add your logo to the issue as an attachment - SVG, JPEG or PNG, and less than 10MB please
```

Reply:
Actually I'd prefer SVG over anything else, and ideally a version that renders well on a dark background. Most companies should have versions of their logo for bright or dark backgrounds, but who knows. I'd be happy to reach out to their marketing depts. in case it's not working with what they provide.